Mlem/Views/Tabs/Posts/Components/Comment/Components/Comment Item.swift

```diff
@@ -179,6 +180,9 @@ struct CommentItem: View {
             } else if !isCollapsed {
                 MarkdownView(text: hierarchicalComment.commentView.comment.content, isNsfw: hierarchicalComment.commentView.post.nsfw)
                     .frame(maxWidth: .infinity, alignment: .topLeading)
+                    .contextMenu {
+                        EasyTranslateButton(text: .constant(hierarchicalComment.commentView.comment.content))
+                    }
```

Comment:
CommentItem already has a context menu on it--can this be integrated?

Might be a byproduct of this PR being ancient lol